.gitignore

```diff
@@ -30,6 +30,7 @@ local.properties
 *.apk
 *.iml
 *.jks
+android/gradle.properties
```

Comment:
My first attempt was to use this dynamic key in `android/gradle.properties`, but it didn't seem right to set that file to `.gitignore` as the app wouldn't run without it. `android/local.properties` also takes precedence, and it seems to be designed for sensitive data such as this one.

The reason why I'd like to avoid putting data into `.gradle` is that I have encountered many cases in which I had to nuke that folder due to experimentation with different node modules. I'd like to avoid having to re-create this file every time that happens. :)